docs/gwcs/developer_introduction.rst

```diff
@@ -0,0 +1,204 @@
+.. _dev_intro:
+
+Introduction to Constructing Generalized World Coordinate System Models
+=======================================================================
+
+Pixel Conventions and Definitions
+---------------------------------
+
+See :ref:`pixel-conventions-and-definitions` for an overview.
+
+Developer Installation
```

Comment:
This feels out of place here, the rest of this doc is about creating a gwcs not contributing to the repo.

Reply:
I agree. I've removed it.